pkg/server/config.go

```diff
@@ -34,9 +34,17 @@ type UIConfig struct {
 	OrganizationTypes []string `yaml:"organization_types" mapstructure:"organization_types"`
 }
 
+type ConnectHeader struct {
```

Comment:
should we move these to session configurations? As these do not affect the connect protocol/server in any way.